policy/SuprimecamMapper.yaml

```diff
@@ -497,3 +497,8 @@ datasets:
     python: hsc.pipe.tasks.multiband.MultiBandConfig
     storage: ConfigStorage
     template: config/multiband.py
+
+  jointcal_wcs:  # astrometric calibration produced by jointcal/meas_mosaic
+    template: jointcal-results/%(filter)s/%(tract)04d/jointcal_wcs-%(visit)07d-%(ccd)03d.fits
+  jointcal_photoCalib:  # photometric calibration produced by jointcal/meas_mosaic
+    template: jointcal-results/%(filter)s/%(tract)04d/jointcal_photoCalib-%(visit)07d-%(ccd)03d.fits
```

Comment:
Don't need the extra space, and I think it makes more sense to include these just before the coadd definitions (i.e. at line ~312...but I'm not insisting!)